src/thor/dijkstras.cc

```diff
@@ -41,8 +41,7 @@ namespace thor {
 
```

Comment:
the gist here is remove all time tracking (except that which is special to the multimodal schedule finding algorithm) and pass around the TimeInfo struct instead